### PR TITLE
[14.0][MIG] barcodes_gs1_label

### PR DESCRIPTION
**Pull request by MiquelRForgeFlow**
Standard migration.

Depends on:
 - [x] https://github.com/nuobit/odoo-addons/pull/23
 - [x] https://github.com/nuobit/odoo-addons/pull/24

### CODE REVIEW

**Review by ChrisOForgeFlow — APPROVED** [review]
Functional and Code Review

File: barcodes_gs1_label/static/src/less/styles.less [hunk 1]
```diff
@@ -0,0 +1,364 @@
+//// GLOBAL
```
Comment (LoisRForgeFlow) [inline barcodes_gs1_label/static/src/less/styles.less]: @MiquelRForgeFlow can you convert to .scss?

Comment (MiquelRForgeFlow) [reply barcodes_gs1_label/static/src/less/styles.less]: done

File: barcodes_gs1_label/report/report_barcode_gs1.py [hunk 2]
```diff
@@ -0,0 +1,187 @@
...
+        if diff:
+            sub_li += [None] * diff
+    yield sub_li
+    yield from chunks(li[n:], n, padding=padding)
+
+
+class ReportGS1Barcode(models.AbstractModel):
+    _name = "report.barcodes_gs1_label.report_gs1_barcode"
+
+    @api.model  # noqa: C901
```
Comment (MiquelRForgeFlow) [inline barcodes_gs1_label/report/report_barcode_gs1.py]: it's rare this noqa is not working in the pre-commit :( 🤔

**Comment by MiquelRForgeFlow** [comment]
travis fails due to https://github.com/OCA/reporting-engine/issues/592

**Comment by MiquelRForgeFlow** [comment]
Updated translation

**Pull request merged**